src/main/java/nextstep/qna/domain/Answers.java

```diff
@@ -0,0 +1,30 @@
+package nextstep.qna.domain;
+
+import nextstep.users.domain.NsUser;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Answers {
+    private List<Answer> answerList;
+
+    public Answers() {
+        this.answerList = new ArrayList<>();
+    }
+
+    public void add(Answer answer) {
+        answerList.add(answer);
+    }
+
+    public List<DeleteHistory> delete() {
```

Comment:
```suggestion
    public List<DeleteHistory> delete(NsUser loginUser) {
```

NsUser를 인자로 넘겨 owner 체크도 이곳에서 함께 하는 것은 어떨까?